### ISSUE TITLE
Changing WebUI Host Port

### ISSUE THREAD

**Issue by RCCRAFT1** [bettercap/bettercap#986]
*Description of the bug or feature request*
Conflicting ports on my network

### Environment

Please provide:

* Bettercap version you are using ( `bettercap -version` ). 
latest

* OS version and architecture you are using. 
Ubuntu 22.04 LTS

* Go version if building from sources. 
N/A

* Command line arguments you are using.
sudo bettercap -caplet http-ui

* Caplet code you are using or the interactive session commands.
http-ui

* **Full debug output** while reproducing the issue ( `bettercap -debug ...` ).
https://i.e-z.host/6tajsi1h.png

### Steps to Reproduce

1. *First Step*
run the VM on hyper-v
3. *Second Step*
get error

**Expected behavior:** *What you expected to happen*
I expect the HTTP UI to start but I have a conflicting address/port and I want to change that

**Actual behavior:** *What actually happened*
Tells me my address is already in use

Basically, I just want to change the localhost address and port. 

-- 

**♥ ANY INCOMPLETE REPORT WILL BE CLOSED RIGHT AWAY ♥**

**Comment by k0i** [comment]
Hello!
I configured that I can change api.rest.port  by editing  `/usr/local/share/bettercap/caplets/http-ui.cap`

```
# api listening on http://127.0.0.1:8081/ and ui to http://127.0.0.1
set api.rest.address 127.0.0.1

# change this value
set api.rest.port 8081

set http.server.address 127.0.0.1
set http.server.port 80
# default installation path of the ui
set http.server.path /usr/local/share/bettercap/ui

# !!! CHANGE THESE !!!
set api.rest.username koyam
set api.rest.password koyam

# go!
api.rest on
http.server on
```

If this is not the information you want.,I'm sorry.Please just ignore it.
